.github/workflows/tox.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python: [3.5, 3.6, 3.8]
```

Comment:
this is at odds with the idea of supporting the Python versions that come with Ubuntu LTS releases (since we always run on 20.04 with differing versions)

Reply:
Well it doesn't work, never had, so nothing is dropped here. It is fixed with a follow-up pr